pulsar/pulsar.sh

```diff
@@ -0,0 +1,225 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+
+readonly ROLE="$(/usr/share/google/get_metadata_value attributes/dataproc-role)"
+readonly CLUSTER_NAME="$(/usr/share/google/get_metadata_value attributes/dataproc-cluster-name)"
+readonly NUM_WORKERS=$(/usr/share/google/get_metadata_value attributes/dataproc-worker-count) 
+readonly ZOOKEEPER_HOME=/usr/lib/zookeeper
+
+readonly PULSAR_FUNCTIONS_ENABLED="$(/usr/share/google/get_metadata_value attributes/pulsar-functions-enabled || echo 'false')"
+readonly BUILTIN_CONNECTORS_ENABLED="$(/usr/share/google/get_metadata_value attributes/builtin-connectors-enabled || echo 'false')"
+readonly TIERED_STORAGE_OFFLOADERS_ENABLED="$(/usr/share/google/get_metadata_value attributes/tiered-storage-offloaders-enabled || echo 'false')"
+
+WEB_SERVICE_URL_PORT="8080" #default value, not recommended to change
```

Comment:
Why are they not recommended to change? Can we allow users to change them with metadata?

Reply:
These port values map to best-practice port assignment as defined by the pulsar docs. see [here](https://pulsar.apache.org/docs/en/deploy-bare-metal/) and cmd+f "you had better not use a different port." We can certainly allow users to change these port values, but I'm not sure the value in doing so if the pulsar docs ward away from doing so. Thoughts?